card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  font-size: var(--description-font-size);
+  font-family: var(--font-family);
+  color: var(--outline-grey);
+  /* border: 1px solid var(--outline-grey); */
+  padding: 40px;
+  gap: 20px;
+  /* border-radius: 15px; */
+}
+
+.content-section {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.name-section {
+  margin-top: 20px;
+}
+
+.name-section-worker {
+  font-size: var(--base-font-size);
+  font-family: var(--font-family);
+  color: var(--black);
+  font-weight: var(--font-bold);
+  padding-right: 10px;
+}
+
+.star {
+  padding-right: 10px;
+}
+
+.star-span {
+  text-decoration: underline dotted 1px;
+  text-underline-offset: 5px;
+}
+
+.price {
+  color: var(--purple);
+  cursor: pointer;
+}
+
+.line-between {
+  background-image: linear-gradient(
+    to right,
+    rgb(227, 227, 227) 40%,
+    rgba(255, 255, 255, 0) 0%
+  );
+  width: 100%;
+  height: 1px;
+  background-size: 5px 1px;
+  background-repeat: repeat-x;
+  background-position: center top;
+}
+
+.worker-pic {
+  width: 10em;
+  border-radius: 15px;
+}
+
+.ImgBtn-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.btn-abs {
+  position: absolute;
+  bottom: -15px;
+}
+
+button {
+  padding: 10px;
+  font-size: var(--button-font-size);
+}
+
+.view-details {
+  color: var(--purple);
+  cursor: pointer;
+}
+
+.line {
+  background-color: var(--outline-grey);
+  height: 1px;
+  width: 100%;
```

Comment:
width default is 100%

Reply:
As the width was not set and the component was not displayed we add a width mannually